### PR TITLE
Backport Release 10.1.3

### PR DESCRIPTION
**Pull request by botanical**
**Summary:** Summary of changes

Addresses [CUMULUS-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-XXX)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -106,7 +106,7 @@
     "eslint-config-airbnb-typescript": "^14.0.1",
     "eslint-plugin-eslint-comments": "^3.0.1",
     "eslint-plugin-import": "2.22.1",
-    "eslint-plugin-jsdoc": "^22.1.0",
+    "eslint-plugin-jsdoc": "^38.1.6",
```
Comment (botanical) [inline package.json:109]: Updated to be compatible with npm v8.6

File: packages/api/ecs/async-operation/tests/test-index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const test = require('ava');
-const uuidv4 = require('uuid/v4');
+const { v4: uuidv4 } = require('uuid');
```
Comment (botanical) [inline packages/api/ecs/async-operation/tests/test-index.js:4]: Was previously getting error
```
[Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './v4' is not defined by "exports"](https://stackoverflow.com/questions/62549457/error-err-package-path-not-exported-package-subpath-v4-is-not-defined-by)
``` 
so I changed the import syntax

File: package.json (hunk 1 above)
Comment (npauzenga) [reply package.json:109]: I think this is fine. If it was a production package that might be different but it seems OK.

File: package.json [hunk 3]
```diff
@@ -127,7 +127,7 @@
     "markdownlint-cli": "^0.19.0",
     "md5": "^2.2.1",
     "mime-types": "^2.1.22",
-    "moment": "2.24.0",
+    "moment": "2.29.2",
```
Comment (npauzenga) [inline package.json:130]: Was this for an earlier vulnerability (pre-2.29.4?)

File: packages/api/ecs/async-operation/tests/test-index.js (hunk 2 above)
Comment (npauzenga) [reply packages/api/ecs/async-operation/tests/test-index.js:4]: Weird that that would have caused an issue here but that syntax is totally fine so 🤷

**Review by npauzenga — APPROVED** [review]
Looks good @botanical!

**Pull request merged**